### PR TITLE
adjustments + bug fixes

### PR DESCRIPTION
**Pull request by asbyth**
remove utilities section
move togglesprint to new section
move victory royale to new section
add better f1 (hide names when in f1, located in vanilla enhancements)
add some breaks in the ingame menu
remove auto optimizer
fix 1.8 debug being broken
remove wildcards
move autotip's config settings to Settings

### CODE REVIEW

**Comment by asbyth** [comment]
shut up i'll fix it error rdil's pr literally got accepted as i was pushing

**Comment by RDIL** [comment]
hah lmfao 
## ALSO CREDIT TO [WESJD](https://wesjd.net) FOR MAKING THE NONAMETAGS MOD

File: src/main/java/cc/hyperium/config/Category.java [hunk 1]
```diff
@@ -2,21 +2,21 @@
 
 public enum Category {
 
-    //Settings Tab
+    // Settings Tab
     GENERAL("General"),
     IMPROVEMENTS("Improvements"),
     INTEGRATIONS("Integrations"),
     COSMETICS("Cosmetics"),
+    CUSTOMIZATION("Customization"),
```
Comment (RDIL) [inline src/main/java/cc/hyperium/config/Category.java]: Whats the point of this tab?

File: src/main/resources/assets/hyperium/lang/af_ZA.lang [hunk 2]
```diff
@@ -227,6 +221,12 @@ gui.settings.mainmenuserver=Main Menu Server
 gui.settings.disable_dances=Disable Dances
 gui.settings.victoryroyale=Victory Royale
 gui.settings.hyperiumprefix=Show Hyperium Chat Prefix
+gui.settings.betterf1=Hide names in F1
```
Comment (RDIL) [inline src/main/resources/assets/hyperium/lang/af_ZA.lang]: You should prob. add to the README

File: src/main/java/cc/hyperium/config/Category.java (hunk 1 above)
Comment (asbyth) [reply src/main/java/cc/hyperium/config/Category.java]: removed it in a different pr, but its so i could get all the customization stuff out of misc bc it was getting packed

File: src/main/resources/assets/hyperium/lang/af_ZA.lang (hunk 2 above)
Comment (asbyth) [reply src/main/resources/assets/hyperium/lang/af_ZA.lang]: will do

File: src/main/java/cc/hyperium/config/Category.java (hunk 1 above)
Comment (RDIL) [reply src/main/java/cc/hyperium/config/Category.java]: oh ok

File: src/main/java/cc/hyperium/Hyperium.java [hunk 3]
```diff
@@ -19,11 +19,38 @@
 
 import cc.hyperium.addons.InternalAddons;
 import cc.hyperium.commands.HyperiumCommandHandler;
-import cc.hyperium.commands.defaults.*;
```
Comment (RDIL) [inline src/main/java/cc/hyperium/Hyperium.java]: really

Comment (asbyth) [reply src/main/java/cc/hyperium/Hyperium.java]: begone wildcards!

Comment (RDIL) [reply src/main/java/cc/hyperium/Hyperium.java]: nO

**Pull request closed without merging**